trulens_eval/trulens_eval/util.py

```diff
@@ -40,6 +41,51 @@
 UNICODE_CHECK = "✅"
 UNCIODE_YIELD = "⚡"
 
+# Optional requirements.
+
+REQUIREMENT_LLAMA = (
+    "llama_index module is required for instrumenting llama_index apps. "
```

Comment:
Do we want to suggest a minimum supported version here?

Reply:
Added suggestion but am not checking for it.